releases/release-1.32/release-notes/maps/pr-121206-map.yaml

```diff
@@ -0,0 +1,17 @@
+pr: 121206
+releasenote:
+  text: |-
+    `StartupProbe` is now stopped explicity when `successThrethold` is reached.
+    This eliminates the problem that `StartupProbe` is executed more than `successThreshold`.
+  documentation: []
```

Comment:
Just a note, when you're editing the release note you just need to uncomment the pr and release notes text (for example: https://github.com/kubernetes/sig-release/blob/97ee888e36d5d50413b6a9290b8d276b7a43e643/releases/release-1.32/release-notes/maps/pr-119089-map.yaml). We should clarify this in the handbook! https://github.com/kubernetes/sig-release/blob/master/release-team/role-handbooks/release-notes/editing-flow.md

Reply:
hi @npolshakova want to confirm with you : when you say 'uncomment' does that mean I can remove the '|-' and also move the text up for a row ?

Reply:
Removing the # from the release notes lines and PR.

Reply:
To clarify, you don't need to change anything in this PR. But for future reference, we don't need to uncomment all the fields when running `krel release-notes --fix`. Maybe it might help to clarify which fields should be uncommented out in the handbook!